src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php

```diff
@@ -136,7 +140,7 @@ public function exists(CanResetPasswordContract $user, $token)
      */
     protected function tokenExpired($createdAt)
     {
-        return Carbon::parse($createdAt)->addSeconds($this->expires)->isPast();
+        return app(Factory::class)->parse($createdAt)->addSeconds($this->expires)->isPast();
```

Comment:
This just to check if the token has expired. Why do we need to make it complicated?